### PR TITLE
Next.js PR 14: Migrate the "Overview" and "Guide" pages to Next.js

### PR DESCRIPTION
**Pull request by danoc**

### CODE REVIEW

File: next/pages/guide/product/breakpoints.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                        <TD>Large</TD>
+                        <TD>
+                            <Code>1025px</Code>
+                        </TD>
+                    </tr>
+                </tbody>
+            </Table>
+
+            <Code language="scss">
+                {`
```
Comment (lavelle) [inline next/pages/guide/product/breakpoints.tsx:110]: Is this how Prettier formats this? I feel like the indentation could be clearer. nbd if you're replacing it with a new code component though.

File: next/pages/guide/product/design-tokens.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import type { InferGetStaticPropsType } from 'next';
+import { ContentPage } from '../../../components/mdx/mdx';
+import getContentPagteStaticProps from '../../../utils/get-content-page-static-props';
+import { Code, Img, LI, P, UL } from '../../../components/mdx/components';
+
+import tokensImage from '../../../images/pages/guide/product/design-tokens/tokens.png';
+
+export const getStaticProps = getContentPagteStaticProps;
```
Comment (lavelle) [inline next/pages/guide/product/design-tokens.tsx:9]: `getContentPagteStaticProps` is a typo

**Review by lavelle — COMMENTED** [review]
Generally i'm a bit confused about how we're using MDX components like `P` or `H2` inside regular TSX files, but I'll trust your plan and assume this is just temporary for the migration away from MDX

File: next/pages/guide/product/breakpoints.tsx (hunk 1 above)
Comment (danoc) [reply next/pages/guide/product/breakpoints.tsx:110]: Yeah, IIRC Prettier is a bit weird about the formatting.

**Review by lavelle — APPROVED** [review]

**Pull request merged**